src/scss/components/_navbar.scss

```diff
@@ -118,13 +117,26 @@ $header-height: 15;
 }
 
 .usa-logo__img {
-  height: units(3);
+  @include at-media-max($theme-navigation-width) {
+    height: units(2);
+  }
+  @include at-media($theme-navigation-width) {
+    height: units(3);
+  }
 }
 
-.usa-logo-text {
-  @include u-font-size($theme-header-font-family, 4);
+.usa-logo__text {
+  @include at-media-max($theme-navigation-width) {
+    @include u-font-size($theme-header-font-family, 3);
+    padding-left: 17px;
+  }
+
+  @include at-media($theme-navigation-width) {
+    @include u-font-size($theme-header-font-family, 4);
+    padding-left: 27px;
```

Comment:
Kinda magic number usage here but we're trying to position text against an image where the logo and text in the logo is all one image. We should eventually re-build this logo in a more sustainable way